CONTRIBUTING.md

```diff
@@ -16,13 +16,14 @@ Want to help? Here's how. Please be sure to check our [Content Style Guide](http
     - [Events](#events)
     - [Patterns](#patterns)
     - [Resources](#resources)
-    - [Promotions, partnership deals, and SEO scams](#promotions-partnership-seals-and-seo-scams)
+    - [Promotions, partnership deals, and SEO scams](#promotions-partnership-deals-and-seo-scams)
```

Comment:
![Animated gif of a seal lying on its back in the snow slapping its belly.](https://media3.giphy.com/media/mpAJq0BoNZbig/giphy.gif?cid=790b76115cbd00164a78593832be83b5)

Reply:
are you suggesting we seal the deal?